src/components/dev-hub/byline-block.js

```diff
@@ -68,15 +68,20 @@ const ByLine = styled('div')`
     }
 `;
 
-const BylineBlock = ({
-    authorName = '',
-    authorImage = DEFAULT_AUTHOR_IMAGE,
-}) => {
+const BylineBlock = ({ authorImage, authorName = '' }) => {
+    const hasImage = !!authorImage;
     const authorLink = `/author/${getTagPageUriComponent(authorName)}`;
     return (
         <ByLine>
             <AuthorImage>
-                <img src={withPrefix(authorImage)} alt={authorName} />
+                <img
+                    src={
+                        hasImage
```

Comment:
Super nit: move this logic to replace hasImage

`const authorImageSrc = !!authorImage ? withPrefix(authorImage) : DEFAULT_AUTHOR_IMAGE;`

Currently it feels like we're doing the image check twice